lua-docs/content/reference/Map.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Particubes
+keywords: particubes, game, mobile, scripting, cube, voxel, world
+---
+
+# Map
+
+`Map` is a global variable that represents the world map.
+
+## Fields
+
+### Width (Integer, read-only)
+
+Returns the map's width, measured in cubes.
+
+### Height (Integer, read-only)
+
+Returns the map's height, measured in cubes.
+
+### Depth (Integer, read-only)
+
+Returns the map's depth, measured in cubes.
+
+### Scale (Float, read-only)
+
+Returns the map's scale factor, measured in pixels per cube.
```

Comment:
You should only say `scale factor`. `scale == 1` doesn't mean `1 cube == 1 pixel`.

Reply:
Right, I called this unit "pixel" in the implementation as I could not find a better name.
Changing this.